Source/Fuse.Controls.Native/Android/ImageView.uno

```diff
@@ -72,22 +72,34 @@ namespace Fuse.Controls.Native.Android
 	extern(Android) internal class ImageView : View, IImageView
 	{
 
+		ImageSource _imageSource;
 		public ImageSource ImageSource
 		{
 			set
 			{
 				if (value == null)
 					ImageHandle = null;
-				else if (value is FileImageSource)
-					UpdateImage((FileImageSource)value);
-				else if (value is HttpImageSource)
-					UpdateImage((HttpImageSource)value);
-				else if (value is MultiDensityImageSource)
-					UpdateImage((MultiDensityImageSource)value);
-				else
-				{
-					throw new Exception(value + " not supported in native context");
-				}
+				_imageSource = value;
+				OnImageSourceChanged();
+			}
+			private get
+			{
+				return _imageSource;
+			}
+		}
+
+		void OnImageSourceChanged()
+		{
+			SourceChanged;
+
+				var mds = (MultiDensityImageSource)ImageSource;
```

Comment:
Soemthing with the spacing is wrong here.

Reply:
There's a chunk of code missing too it seems, when compared to the iOS code.

Reply:
yeah, no idea how i missed that, but that's indeed what happened :S